### PR TITLE
fix(JobListing): date posted get stacked

### PR DESCRIPTION
**Pull request by shubhamku044**
### What is this PR solving?
Fixed the information which was getting stacked earlier.

### Screenshots/Video
image

### Testing instructions
Tested for all the screen sizes.

### Confirm 

 [ ] I tested and confirmed the changes
 [ ] This PR is safe to rollback.
 [ ] Added the correct label
 [ ] This PR follows the recommendations regarding squashig commits

### CODE REVIEW

File: src/components/JobListings/components/JobListing/JobListing.tsx [hunk 1]
```diff
@@ -29,7 +29,7 @@ export function JobListing({ job, addFilter, removeFilter, filters }: Props) {
         alt="logo"
         className="-mt-[25px] md:mt-0 w-12 md:w-[80px] md:h-[80px]"
       />
-      <div>
+      <div className="min-w-[225px]">
```
Comment (jkeohan) [inline src/components/JobListings/components/JobListing/JobListing.tsx]: Notced that there are still some elements that stack with min-w-[225px]...I experimented with some settings and found that flex-[0_1_120%] worked better...
Screen Shot 2023-01-31 at 7 06 55 PM

This also required that I make some additional changes like removing the hr on md above and padding around the tag wrapper

Comment (jkeohan) [reply src/components/JobListings/components/JobListing/JobListing.tsx]: If you update the div and replace min-w with ` flex-[0_1_120%]` and make the additional changes in this branch I just pushed I'll approve your PR...https://github.com/jkeohan/fe-mentor-job-listings/tree/bug-21-dates-stacked

Comment (shubhamku044) [reply src/components/JobListings/components/JobListing/JobListing.tsx]: I have made the necessary changes, you can check.

**Review by jkeohan — APPROVED** [review]
changes look good but I was wrong on the 120% as I didn't test desktop at 1440px...it should be 100% but I'll make the edit in another pr and approve this one

**Pull request merged**